src/relay/libp2p_transport_relay.erl

```diff
@@ -130,10 +130,10 @@ check_peerbook(TID, MAddr) ->
 -spec connect_rcv(pid(), string(), string(), pid()) -> {ok, pid()} | {error, term()}.
 connect_rcv(Swarm, MAddr, SAddress, SessionPid) ->
     receive
```

Comment:
do we know the pid of the process that we're expecting a message from here?  if possible, can we monitor is so that if it's dead we'll know right away instead of blocking for 15s?